static_src/constants.js

```diff
@@ -260,10 +260,16 @@ const userActionTypes = keymirror({
   USER_INVITE_STATUS_UPDATED: null,
   // Action to trigger when user list notice is created.
   USER_LIST_NOTICE_CREATED: null,
-  // Action to trigger email sent to user with cloud.gov invite url.
+  // Action to trigger email sent to user with cloud.gov invite url
+  // and associate user to organization.
   USER_ORG_ASSOCIATE: null,
-  // Action to associate user to organization on the server.
+  // Action when user is associated to organization.
   USER_ORG_ASSOCIATED: null,
+  // Action to trigger email sent to user with cloud.gov invite url
+  // and associate user to space.
+  USER_SPACE_ASSOCIATE: null,
```

Comment:
I think this comment is slightly misleading, since this action type doesn't trigger the email invite as a result of it being dispatched.